docs/contribute/benchmarks/TRAINING/benchmarks_DL2_EnergyLUT.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Should plot energy bias here as well (just the mean of the distribution at each true energy)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/protopipe/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>DONE</p>